server/src/main/java/keywhiz/service/daos/SecretSeriesDAO.java

```diff
@@ -131,8 +152,7 @@ void updateSecretSeries(
     }
 
     try {
-      String rowHmac = rowHmacGenerator.computeRowHmac(
-          SECRETS.getName(), List.of(name, secretId));
+      String rowHmac = computeRowHmac(secretId, name);
```

Comment:
The logic to compute the HMAC is repeated in a couple places so I introduced a helper method to reduce duplication.